wgpu/src/lib.rs

```diff
@@ -1936,12 +1961,34 @@ impl Instance {
     /// - On macOS/Metal: will panic if not called on the main thread.
     /// - On web: will panic if the `raw_window_handle` does not properly refer to a
     ///   canvas element.
-    pub unsafe fn create_surface<
-        W: raw_window_handle::HasWindowHandle + raw_window_handle::HasDisplayHandle,
+    pub fn create_surface<
+        'window,
+        W: HasWindowHandle + HasDisplayHandle + WasmNotSend + WasmNotSync + 'window,
```

Comment:
Still need to move these to where clasues.

Reply:
Done!
Applied the same to `create_surface_from_raw()` as well.